meta/meta.go

```diff
@@ -0,0 +1,343 @@
+	}
+	content, err := http.Get("https://github.com/dasmeta/meta/releases/download/v0.1.1/package.json")
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer content.Body.Close()
+
+	io.Copy(file, content.Body)
+	fmt.Println("package.json file creation is done")
+}
+
+func createCommitLintConfigFile() {
+
+	file, err := os.Create("commitlint.config.js")
+
+	defer file.Close()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	content, err := http.Get("https://github.com/dasmeta/meta/releases/download/v0.1.1/commitlint.config.js")
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer content.Body.Close()
+
+	io.Copy(file, content.Body)
+	fmt.Println("commitlint.config.js file creation is done")
+}
+
+func createGithubFolder() {
+	if err := os.Mkdir(".github", os.ModePerm); err != nil {
+		log.Fatal(err)
+	}
+
+	os.Chdir(".github")
+	fmt.Println("create .github folder")
+	createWorkflowFolder()
+	createCommitLintYamlFile()
+	os.Chdir("../../")
+
+}
+
+func createWorkflowFolder() {
+	if err := os.Mkdir("workflows", os.ModePerm); err != nil {
+		log.Fatal(err)
+	}
+	os.Chdir("workflows")
+	fmt.Println("create workflows folder")
+}
+
+func createCommitLintYamlFile() {
+	file, err := os.Create("commitlint.yaml")
+
+	defer file.Close()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	content, err := http.Get("https://github.com/dasmeta/meta/releases/download/v0.1.1/commitlint.yaml")
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer content.Body.Close()
+
+	io.Copy(file, content.Body)
+	fmt.Println("commitlint.yaml file creation is done")
+
+}
+
+func createHuskyFolder() {
+	if err := os.Mkdir(".husky", os.ModePerm); err != nil {
+		log.Fatal(err)
+	}
+	os.Chdir(".husky")
+	fmt.Println("create .husky folder")
+	createCommitMSGFile()
+	create_Folder()
+	os.Chdir("_")
+}
+
+func create_Folder() {
+	if err := os.Mkdir("_", os.ModePerm); err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println("create _ folder")
+	createHuskyshFile()
+	createGitIgnoreFile()
+}
+
+func createHuskyshFile() {
+	file, err := os.Create("husky.sh")
+
+	defer file.Close()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	content, err := http.Get("https://github.com/dasmeta/meta/releases/download/v0.1.1/husky.sh")
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer content.Body.Close()
+
+	io.Copy(file, content.Body)
+	fmt.Println("husky.sh file creation is done")
+
+}
+
+func createCommitMSGFile() {
+	file, err := os.Create("commit-msg")
+
+	defer file.Close()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	content, err := http.Get("https://github.com/dasmeta/meta/releases/download/v0.1.1/commit-msg")
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer content.Body.Close()
+
+	io.Copy(file, content.Body)
+	fmt.Println("commit-msg file creation is done")
+
+}
+
+func createGitIgnoreFile() {
+	file, err := os.Create(".gitignore")
+
+	defer file.Close()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	content, err := http.Get("https://github.com/dasmeta/meta/releases/download/v0.1.1/default.gitignore")
```

Comment:
lets have some constant/global-variable for version so that instead having v0.1.1 version in 6 places we will have it set in one place as variable and the variable will be used in those 6 places,
this will allow more easily update this file with new changes, so that we will increase package version in on central place